src/Jackalope/NodeType/NodeProcessor.php

```diff
@@ -193,7 +209,7 @@ private function processNodeWithType(NodeInterface $node, NodeType $nodeTypeDefi
                                 $value = $defaultValues;
                             } elseif (isset($defaultValues[0])) {
                                 $value = $defaultValues[0];
-                            } else {
+                            } elseif ($nodeTypeDefinition->getName() !== VersionHandler::MIX_VERSIONABLE) {
```

Comment:
we now call the version handler before this, so i think we can remove this check and go back to just `else`, no?